README.reproducible.md

```diff
@@ -0,0 +1,148 @@
+#Armory Reproducible Build Documentation
+
+##Gitian
+
+Gitian is the build system that Bitcoin Core uses for reproducible builds.
+Armory has adopted it for OS X and Windows reproducible builds. This guide
+assumes you are using KVM as the virtualization technology to make the VMs
+that Gitian uses to build in, but gitian-tools also supports LXC.
```

Comment:
Sure, I just wanted to keep things simple and include an explanation of what I use, which is KVM. People should be able to use whichever one they want to use and the result should still be reproducible, even if someone is using the other one.

It's actually just a matter of an environment variable and an extra flag, so I'll document it.